conf/messages.conf

```diff
@@ -289,7 +289,8 @@
 287: You cannot change party leaders in this map.
 //Missing stuff for @killer related commands.
 288: You are no longer killable.
-//289-290 FREE
+289: Target character must be in the same map.
+//290 FREE
```

Comment:
Why `20120410`? The packet was added in `20060424` so I don't think we even need to check version at all